jscomp/test/record_regression.res

```diff
@@ -3,10 +3,12 @@
 @obj
 type t0 = {x: int, y: option<int>, z: int}
 
-// let f0 = { x : 3 ,y : None}
-
 let f1 = {x: 3, z: 2}
 
+let f2 = {x: 3, z: 3, y: 3}
```

Comment:
@mattdamon108 see how this is written now.

Reply:
Wow! this is perfect!